core/src/banking_stage.rs

```diff
@@ -1237,6 +1241,24 @@ impl BankingStage {
             .collect_vec()
     }
 
+    fn mark_accepted_packets_as_forwarded(
```

Comment:
To avoid breaking more encapsulation, I think this should be a method in `UnprocessedPacketBatches`

Reply:
good call, moved it to `UnprocessedPacketBatches`